dts/arm/nxp/nxp_rt11xx.dtsi

```diff
@@ -964,6 +965,7 @@
 			nxp,a_on;
 			reg = <0x40c14000 0x4000>,
 				<0x40c18000 0x4000>;
+			reg-names = "edma", "dmamux0";
```

Comment:
for rt11xx series, dmamux0 is for M7 domain, and dmamux1 is for LPST domain(m4). using dmamux0 is not correct

Reply:
ok, so i will consider to update this

Reply:
I've updated, the dmamux is used is the one that can route DMA sources.